lib/server.js

```diff
@@ -80,6 +144,23 @@ server.get('/tasks/:task/stats', function(req, res, next) {
     .catch(next);
 });
 
+/**
+ * Get a list of items for a task
+ * @name /tasks/:task/items
+ * @param {Object} params - what the keys in the url mean
+ * @param {String} params.task - the task id
+ * @param {Object} query - what queryparams are valid
+ * @param {String} [query.lock] - if provided this must either be 'unlocked' or 'locked'
+ * @example
+ * curl https://host/tasks/task-name/items?lock=unlocked
```

Comment:
Looking back, I should have called the id of the task `task-id` rather than `task-name`. Can we make that change through out the docs?